src/jdk.crypto.ec/share/classes/sun/security/ec/ECPrivateKeyImpl.java

```diff
@@ -71,7 +71,6 @@
 public final class ECPrivateKeyImpl extends PKCS8Key implements ECPrivateKey {
 
     private static final long serialVersionUID = 88695385615075129L;
-    private static final NativeCrypto nativeCrypto = NativeCrypto.getNativeCrypto();
```

Comment:
Why not keep this field like other uses of `NativeCrypto.getNativeCrypto()`? Same question for `ECPublicKeyImpl`.

Reply:
`NativeCrypto.getNativeCrypto()` is going to invoke `initCryptoLibraries()` which loads the native library. This PR delays such initialization until its actual usage which is not expected before a checkpoint.
Similar reason for `ECPublicKeyImpl`.